src/model.py

```diff
@@ -0,0 +1,65 @@
+"""
+Model architecture code.
+
+Code structure adapted from Lightning project seed at
+https://github.com/Lightning-AI/deep-learning-project-template
+"""
+import lightning as L
+import torch
+
+
+# %%
```

Comment:
Why do we have the artifact `%%` before starting a class?

Reply:
These are used to make code blocks in Python scripts, and are supported by a few editors, see https://jupytext.readthedocs.io/en/latest/formats-scripts.html#the-percent-format.